build/gulpfile.reh.js

```diff
@@ -476,7 +477,8 @@ function tweakProductForServerWeb(product) {
 
 			const serverTask = task.define(`vscode-${type}${dashed(platform)}${dashed(arch)}${dashed(minified)}`, task.series(
 				compileBuildTask,
-				compileExtensionsBuildTask,
+				cleanExtensionsBuildTask,
+				compileNonNativeExtensionsBuildTask,
 				compileExtensionMediaBuildTask,
```

Comment:
Don't we want `compileExtensionsBuildTask` here ? I believe this task is only used for local build, so useful to build all targets ?

Reply:
It does build all targets. We build the non-native ones here, and then the native ones in the `serverTaskCI` which is included in this local build task at the bottom

Reply:
Sorry missed it